tests/expected/TileBBox.sql.out

```diff
@@ -3,9 +3,9 @@
  POLYGON((-20037508.34 20037508.34,-20037508.34 -20037508.34,20037508.34 -20037508.34,20037508.34 20037508.34,-20037508.34 20037508.34))
 (1 row)
 
-                                                             v2                                                             
-----------------------------------------------------------------------------------------------------------------------------
- POLYGON((-8590298.99 4715858.9,-8590298.99 4696291.02,-8570731.11 4696291.02,-8570731.11 4715858.9,-8590298.99 4715858.9))
+                                                                     v2                                                                     
```

Comment:
i don't think this test is relevant to the pull request - did you mix two PRs together?

Reply:
Without rebuild expected it returns `Files build/TileBBox.sql.out and tests/expected/TileBBox.sql.out differ`